### PR TITLE
[fluid remove]: remove paddle.fluid.layers.yolo_box and paddle.fluid.layers.yolov3_loss

### PR DESCRIPTION
**Pull request by zhengqiwen1997**
### PR types

Others
### PR changes

APIs
### Describe

To clean the code in paddle.fluid, the useless codes should be removed and code still be used should move to a new position.

For function paddle.fluid.layers.yolo_box and paddle.fluid.layers.yolov3_loss, since it is not used in version 2.x, should be removed.

For unittest code where this function is used, if it is aimed to test the old API, it can be removed directly. Otherwise, it should be replaced with paddle.vision.ops.yolo_box and paddle.vision.ops.yolo_loss.

### CODE REVIEW

**Review by zoooo0820 — DISMISSED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/xpu/test_device_guard_xpu.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def test_cpu_only_op(self):
             anchor_mask = [0, 1, 2]
             with paddle.static.device_guard("xpu"):
                 # yolov3_loss only has cpu kernel, so its cpu kernel will be executed
```
Comment (jeff41404) [inline python/paddle/fluid/tests/unittests/xpu/test_device_guard_xpu.py]: `yolov3_loss` in annotation should also be changed

File: python/paddle/fluid/tests/unittests/test_device_guard.py [hunk 2]
```diff
@@ -128,7 +128,7 @@ def test_cpu_only_op(self):
             anchor_mask = [0, 1, 2]
             with paddle.static.device_guard("gpu"):
                 # yolov3_loss only has cpu kernel, so its cpu kernel will be executed
-                loss = fluid.layers.yolov3_loss(
+                loss = paddle.vision.ops.yolo_loss(
```
Comment (jeff41404) [inline python/paddle/fluid/tests/unittests/test_device_guard.py:131]: `yolov3_loss` in annotation should also be changed

File: python/paddle/fluid/tests/unittests/test_device_guard.py [hunk 3]
```diff
@@ -128,7 +128,7 @@ def test_cpu_only_op(self):
             anchor_mask = [0, 1, 2]
             with paddle.static.device_guard("gpu"):
                 # yolov3_loss only has cpu kernel, so its cpu kernel will be executed
```
Comment (jeff41404) [inline python/paddle/fluid/tests/unittests/test_device_guard.py]: `yolov3_loss` in annotation should also be changed

**Review by zoooo0820 — APPROVED** [review]

**Review by jeff41404 — APPROVED** [review]
LGTM

**Review by Ligoml — APPROVED** [review]
LGTM for docs

**Pull request merged**